rpcs3/main.cpp

```diff
@@ -320,22 +325,22 @@ int main(int argc, char** argv)
 		{
 			if (fs::exists(lock_name))
 			{
-				report_fatal_error("Another instance of RPCS3 is running. Close it or kill its process, if necessary.");
+				report_fatal_error(sstr(QCoreApplication::tr("Main", "Another instance of RPCS3 is running. Close it or kill its process, if necessary.")));
```

Comment:
these translations don't make much sense, as there is no translation before the gui application was properly initialized.

And you swapped the text with a comment anyway 🤦

Reply:
If there is no translation loaded at the time, then it will simply show exactly the text presented, so no degredation.
Not sure what you mean with 'And you swapped the text with a comment anyway'.
QString translate(const char *context, const char *sourceText, const char *disambiguation = nullptr, int n = -1)
The context of 'Main' isn't ideal... but it's the main() function, so it's better than nothing.

Reply:
static QString tr(const char *sourceText, const char * = nullptr, int = -1)

Reply:
You're looking at the wrong signature.
You've provided the signature for QObject::tr() not QCoreApplication::tr().

Reply:
static inline QString tr(const char *sourceText, const char *disambiguation = nullptr, int n = -1) \
        { return QCoreApplication::translate(#context, sourceText, disambiguation, n); } \
    QT_DECLARE_DEPRECATED_TR_FUNCTIONS(context) \

Reply:
Ok, so it is my stuff up not using translate, and trusting the Qt statements around tr being implemented in terms of translate.  Sorry to be stubborn on that.